Emerging_Threats/guloader.yml

```diff
@@ -0,0 +1,155 @@
+version: 3
+resources:
+  replicant:
+    - exfil
+Rules:
+  'GuLoader #001':
+    namespace: general
+    detect:
+      event: DNS_REQUEST
+      op: is
+      path: event/DOMAIN_NAME
+      value: ip130.ip-5-39-3.eu
+    respond:
+      - action: report
+        name: guloader-2021-12-02
+  'GuLoader #002':
+    namespace: general
+    detect:
+      event: DNS_REQUEST
+      op: is
+      path: event/DOMAIN_NAME
+      value: nowautomation.com
```

Comment:
Is it just this domain that's bad, or also possibly subdomains?
If subdomains should be included you could to:
```yaml
event: DNS_REQUEST
op: is
path: event/DOMAIN_NAME
value: nowautomation.com
sub domain: '-2:'
```

as documented here: https://doc.limacharlie.io/docs/documentation/ZG9jOjE5OTA0MjQ0-reference-operators#sub-domain

Reply:
https://refractionpoint.slack.com/archives/C02DNHSF8JF/p1638549218167000?thread_ts=1638478408.157200&cid=C02DNHSF8JF